ontology/observable/observable.ttl

```diff
@@ -10637,6 +10781,20 @@ observable:partitionOffset
 	rdfs:range xsd:integer ;
 	.
 
+observable:partyDevice
+	a owl:ObjectProperty ;
+	rdfs:label "partyDevice"@en ;
+	rdfs:comment "The IMEI of the device connected to the cell tower."@en ;
+	rdfs:range observable:Device ;
+	.
+
+observable:partySubscription
+	a owl:ObjectProperty ;
+	rdfs:label "partySubscription"@en ;
+	rdfs:comment "The addresses of the device connected to the cell tower (references IMSI, MSISDN SIP address, 5G supi)."@en ;
```

Comment:
This property will need a demonstration to prevent thing-vs-string representation confusion.